lib/_tls_wrap.js

```diff
@@ -286,6 +286,18 @@ function onnewsession(sessionId, session) {
 }
 
 
+function onkeylogclient(line) {
+  debug('client onkeylog');
+  this[owner_symbol].emit('keylog', line.toString('ascii'));
```

Comment:
toString-ing the buffer still leaves key data on the JS heap, and it sets in stone that the public API is a string, something that would be very difficult to change later on. It would automatically be semver-major because it's a backwards incompatible change. It should be a buffer from the start.

Reply:
I understand your concern now, I've changed the API to Buffer.
Since `crypto.alloc` doesn't exist yet, on the examples I'm still doing `${line}\n`, is this okay?